lib/sse.ts

```diff
@@ -15,9 +15,12 @@ export function sse<
   T extends ServerSentEventMessage,
   TDone extends ServerSentEventMessage,
 >(
-  op: (send: (value: T) => Operation<void>) => Operation<TDone>,
+  op: (
+    request: Request,
+    send: (value: T) => Operation<void>,
+  ) => Operation<TDone>,
 ): Middleware<Request, Response> {
-  return function* () {
+  return function* (request: Request, _next) {
```

Comment:
I'd just get rid of this parameter... no need to hold it if we aren't going to use it and it isn't in front of one that we are.